pinot-controller/src/test/java/org/apache/pinot/controller/helix/core/util/SegmentDeletionManagerTest.java

```diff
@@ -357,6 +358,9 @@ public void createTableAndSegmentFiles(File tempDir, List<String> segmentIds)
     tableDir.mkdir();
     for (String segmentId : segmentIds) {
       createTestFileWithAge(tableDir.getAbsolutePath() + File.separator + segmentId, 0);
+      // Create segment metadata file
+      createTestFileWithAge(
```

Comment:
Is this covering the test for metadata delete?

Reply:
yes added metadata file as well. At the end of the test, there's a verification that the directory does not have any files -

```
try {
        Assert.assertEquals(tableDir.listFiles().length, 0);
        Assert.assertTrue(!deletedTableDir.exists() || deletedTableDir.listFiles().length == ); 
```